bin/templates/scripts/cordova/lib/plugman/pluginHandlers.js

```diff
@@ -91,18 +93,29 @@ var handlers = {
             }
         },
         uninstall:function(obj, plugin, project, options) {
+            var podsJSON = require(project.projectDir + '/pods.json');
```

Comment:
Here, use require(path.join(project.projectDir, 'pods.json')) so it supports windows paths